docs/index.rst

```diff
@@ -1,96 +1,119 @@
+- `Neptune client <https://github.com/neptune-ai/neptune-client>`_ is an open source Python library that allows you to integrate your Python scripts with Neptune. Neptune client supports the following cases:
+
+    - Creating and tracking experiments
+    - Managing running experiment
+    - Querying experiments and projects (search/download)
+
+Get Started
+===========
+
+- New user? `Register <https://neptune.ai/register>`_ and climb aboard.
+- Registered already? Log in `here <https://neptune.ai/login>`_.
+
+
+Track, Organize, Collaborate
+============================
 
 .. image:: ./_static/images/overview/quick_overview.gif
    :target: ./_static/images/overview/quick_overview.gif
    :alt: image
 
-Neptune is lightweight
-----------------------
-Neptune is built with the single design principle in mind: *being lightweight*. What does it mean in practice?
 
-* easy user onboarding: if you know how to use ``print()`` you will learn how to use it in no time.
-* 20-minute deployment: use SaaS, deploy on any cloud or own hardware (|contact us| to learn more).
-* Neptune fits in any workflow, ranging from data exploration & analysis, decision science to machine learning and deep learning.
-* Neptune works with common technologies in data science domain: Python 2 and 3, Jupyter Notebooks, R.
-* Neptune integrates with other tools like |MLflow| and |TensorBoard|.
+The Neptune workflow comprises three iterative phases:
 
-Neptune's focus: track, organize and collaborate
-------------------------------------------------
-.. _track:
+- **Track** all objects in the data science or machine learning project. It can be model training curves, visualizations, input data, calculated features and so on. The snippet below presents an example of integration with Python code.
 
-Track
-^^^^^
-Track all objects in the data science or machine learning project. It can be model training curves, visualizations, input data, calculated features and so on.
-Snippet below, presents example integration with Python code.
+    .. code-block:: bash
 
-.. code-block::
+        import neptune
 
-   import neptune
+        neptune.init('shared/onboarding',
+                    api_token='eyJhcGlfYWRkcmVzcyI6Imh0dHBzOi8vdWkubmVwdHVuZS5tbCIsImFwaV9rZXkiOiJiNzA2YmM4Zi03NmY5LTRjMmUtOTM5ZC00YmEwMzZmOTMyZTQifQ==')
 
-   neptune.init('shared/onboarding',
-                api_token='eyJhcGlfYWRkcmVzcyI6Imh0dHBzOi8vdWkubmVwdHVuZS5tbCIsImFwaV9rZXkiOiJiNzA2YmM4Zi03NmY5LTRjMmUtOTM5ZC00YmEwMzZmOTMyZTQifQ==')
-   with neptune.create_experiment():
-       neptune.append_tag('minimal-example')
-       n = 117
-       for i in range(1, n):
-           neptune.send_metric('iteration', i)
-           neptune.send_metric('loss', 1/i**0.5)
-       neptune.set_property('n_iterations', n)
 
-``api_token`` belongs to the public user |Neptuner|. So, when started you can see your experiment at the top of |experiments view|.
+    .. note::
+       The `api_token` belongs to the public user Neptuner.
```

Comment:
I think another sentence is required here. Something along the lines of:
After running the code, your experiment will appear here: https://ui.neptune.ai/shared/onboarding/experiments